Majasdarbs/05-majasdarbs/Object/cv.js

```diff
@@ -1,3 +1,27 @@
-const VardsUzvards = {
-    
-}
\ No newline at end of file
+const cilveks = {
+    firstName: "Edvards",
+    lastName: "Safronovs",
+    age: 26,
+   
+    skills: {
+        LanguagesSpoken: [
+            "Russian",
+            "Latvian",
+            "English"
+        ],
+        
+    Web:["HTML, CSS, React JS, Bootstrap"],
+
+    },
+
+    Workexperience: "Tele2, Tele3",
+    Education: "Secondary SChool, Latvian Univercity",
+
```

Comment:
you can use formatter, files will look much better. Just a visual stuff, not a mistake :)